app/services/main_conversation/tools/websocket_utils/handle_websocket_connection.py

```
@@ -79,6 +80,7 @@ async def handle_websocket_connection(websocket: WebSocket):
         ).model_dump())
 
         transcriber = TranscriberService()
+        audio_buffer = AudioStreamBuffer()
```

Ruff (0.11.9)

Local variable `audio_buffer` is assigned to but never used

Remove assignment to unused variable `audio_buffer`

(F841)